### ISSUE TITLE
--profile option in carbon-cache does not appear to work

### ISSUE THREAD

**Issue by cswingler** [graphite-project/carbon#450]
I can't get the --profile option to work. 

```
./bin/carbon-cache.py   --profile=test.profile start
Traceback (most recent call last):
  File "./bin/carbon-cache.py", line 32, in <module>
    run_twistd_plugin(__file__)
  File "/opt/graphite/lib/carbon/util.py", line 90, in run_twistd_plugin
    config.parseOptions(twistd_options)
  File "/usr/local/lib/python2.7/dist-packages/twisted/application/app.py", line 579, in parseOptions
    usage.Options.parseOptions(self, options)
  File "/usr/local/lib/python2.7/dist-packages/twisted/python/usage.py", line 237, in parseOptions
    raise UsageError(str(e))
twisted.python.usage.UsageError: option --instance not recognized
```

No combination of `--instance` or other flags seems to make any difference.  Starting the daemon with simply `./bin/carbon-cache.py start` works as expected.

If I'm doing something obviously wrong, the docs probably need to be updated, but I believe this may very well be a bug.

**Comment by deniszh** [comment]
Which version of carbon are you using?
Fix for master is already merged - https://github.com/graphite-project/carbon/pull/422
Will create PR for 0.9.x

**Comment by cswingler** [comment]
Darnit, yeah, I'm still on 0.9.13. Thanks for pointing out the appropriate patch.

**Comment by obfuscurity** [comment]
Merged #451 which should fix this issue.
